### PR TITLE
cloud: update the TiDB Cloud support info in TiDB SQL docs

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added or deleted? (Required)

This PR updates the TiDB Cloud support info in TiDB SQL docs according to https://docs.pingcap.com/tidbcloud/limited-sql-features. After the changes are made, users can easily tell whether they can use one feature on TiDB Serverless or TiDB Cloud.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: mysql-schema.md [hunk 1]
```diff
@@ -55,19 +59,22 @@ Currently, the `help_topic` is NULL.
 
 ## TTL related system tables
 
+> **Note:**
+>
+> This TTL related system tables are not available on [TiDB Serverless](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-serverless) clusters.
```
Comment (qiancai) [inline mysql-schema.md:64]:
```suggestion
> The TTL related system tables are not available on [TiDB Serverless](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-serverless) clusters.
```

File: statistics.md [hunk 2]
```diff
@@ -894,9 +896,17 @@ mysql> show warnings;
...
 * [DROP STATS](/sql-statements/sql-statement-drop-stats.md)
+* [LOCK STATS](/sql-statements/sql-statement-lock-stats.md)
+* [UNLOCK STATS](/sql-statements/sql-statement-unlock-stats.md)
+* [SHOW STATS_LOCKED](/sql-statements/sql-statement-show-stats-locked.md)
 
 </CustomContent>
 
+<CustomContent platform="tidb-cloud">
+
+* [LOAD STATS](/sql-statements/sql-statement-load-stats.md)
```
Comment (sunxiaoguang) [inline statistics.md:907]: Is this line duplicated?

File: statistics.md [hunk 3]
```diff
@@ -717,13 +717,9 @@ The preceding statement only deletes GlobalStats generated in dynamic pruning mo
...
 ## Load statistics
 
-<CustomContent platform="tidb-cloud">
-
 > **Note:**
 >
-> This section is not applicable to TiDB Cloud.
-
-</CustomContent>
+> This feature is not available on [TiDB Serverless](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-serverless) clusters.
```
Comment (sunxiaoguang) [inline statistics.md:722]: Only `load stats` is disabled due to security reasons. Other stats related sql statements are available.

File: sql-statements/sql-statement-show-config.md [hunk 4]
```diff
@@ -7,6 +7,10 @@ summary: Overview of the use of SHOW CONFIG in the TiDB database
 
 The `SHOW CONFIG` statement is used to show the current configuration of various components of TiDB. Note that the configuration and system variables act on different dimensions and should not be mixed up. If you want to obtain the system variable information, use the [SHOW VARIABLES](/sql-statements/sql-statement-show-variables.md) syntax.
 
+> **Note:**
+>
+> This feature is not available on [TiDB Serverless](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-serverless) clusters.
+
```
Comment (sunxiaoguang) [inline sql-statements/sql-statement-show-config.md:13]: It is not available on both serverless and dedicated.

Comment (sunxiaoguang) [reply sql-statements/sql-statement-show-config.md:13]: There might be some other features are not available on both.

File: TOC-tidb-cloud.md [hunk 5]
```diff
@@ -367,6 +370,7 @@
       - [`INSERT`](/sql-statements/sql-statement-insert.md)
       - [`KILL [TIDB]`](/sql-statements/sql-statement-kill.md)
       - [`LOAD DATA`](/sql-statements/sql-statement-load-data.md)
+      - [`LOAD STATS`](/sql-statements/sql-statement-load-stats.md)
```
Comment (sunxiaoguang) [inline TOC-tidb-cloud.md:373]: Is this line duplicated?

File: sql-statements/sql-statement-show-config.md (hunk 4 above)
Comment (qiancai) [reply sql-statements/sql-statement-show-config.md:13]: Thanks for the catch. I will double check the support info for other features again.

File: sql-statements/sql-statement-show-config.md [hunk 6]
```diff
@@ -7,6 +7,10 @@ summary: Overview of the use of SHOW CONFIG in the TiDB database
 
 The `SHOW CONFIG` statement is used to show the current configuration of various components of TiDB. Note that the configuration and system variables act on different dimensions and should not be mixed up. If you want to obtain the system variable information, use the [SHOW VARIABLES](/sql-statements/sql-statement-show-variables.md) syntax.
 
+> **Note:**
+>
+> This feature is not available on [TiDB Serverless](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-serverless) clusters.
```
Comment (qiancai) [inline sql-statements/sql-statement-show-config.md:12]:
```suggestion
> This feature is only applicable to TiDB Self-Hosted and not available on [TiDB Cloud](https://docs.pingcap.com/tidbcloud/). 
```

File: statistics.md (hunk 3 above)
Comment (qiancai) [inline statistics.md:722]:
```suggestion
> Loading statistics is not available on [TiDB Serverless](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-serverless) clusters.
```

File: statistics.md (hunk 2 above)
Comment (qiancai) [reply statistics.md:907]: Do you mean there is one for "tidb" and another for "tidb-cloud"?
image

File: TOC-tidb-cloud.md (hunk 5 above)
Comment (sunxiaoguang) [reply TOC-tidb-cloud.md:373]: Sorry for my mistake. They are different, LOAD vs LOCK

File: statistics.md (hunk 2 above)
Comment (sunxiaoguang) [reply statistics.md:907]: Sorry for my mistake. They are different, LOAD vs LOCK

**Review by sunxiaoguang — APPROVED** [review]
LGTM👍

File: sql-statements/sql-statement-select.md [hunk 7]
```diff
@@ -13,13 +13,9 @@ The `SELECT` statement is used to read data from TiDB.
...
 ![SelectStmt](/media/sqlgram/SelectStmt.png)
 
-<CustomContent platform="tidb-cloud">
-
 > **Note:**
 >
-> The `SELECT ... INTO OUTFILE` statement is not supported by TiDB Cloud.
-
-</CustomContent>
+> The `SELECT ... INTO OUTFILE` statement is only applicable to TiDB Self-hosted and not available on [TiDB Cloud](https://docs.pingcap.com/tidbcloud/).
```
[the remark this answers is not in the document]
Comment (qiancai) [reply sql-statements/sql-statement-select.md]: updated in https://github.com/pingcap/docs/pull/14674/commits/b97326c9606941534152573698fab5b3a5d201bc

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**